static/cypress/integration/task_view.spec.ts

```diff
@@ -38,7 +47,8 @@ describe("TaskView", () => {
       intercept_next_image: intercept_next_image_with_failure,
     });
     login()
-      .wait("@get_task")
+      .wait("@get_tasks")
```

Comment:
Good question, I could not observe any difference in the test runs, so I added the line to the 3 tests where it was missing, for consistency 👍

Reply:
Probably we there already wait for something different that happens afterwards, so that get_tasks is waited for implicitly. But honestly idk, adding it is not gonna break anything so feel free to do it :)